apps/src/code-studio/verifiedTeacherRedux.js

```diff
@@ -1,7 +1,7 @@
 const SET_VERIFIED = 'verifiedTeacher/SET_VERIFIED';
 const SET_VERIFIED_RESOURCES = 'verifiedTeacher/SET_VERIFIED_RESOURCES';
 
-export const setVerified = isVerified => ({type: SET_VERIFIED});
+export const setVerified = () => ({type: SET_VERIFIED});
```

Comment:
it looks like all consumers of this method are already calling it with no args (i.e., `setVerified()` instead of `setVerified(true)`), but can you please confirm?

Reply:
good call, I have double-checked and this is only called as `setVerified()`.